gitman/models/source.py

```diff
@@ -18,6 +24,7 @@ class Source:
     type: str = 'git'
     sparse_paths: List[str] = field(default_factory=list)
     link: Optional[str] = None
+    links: Optional[List[Link]] = field(default_factory=list)
```

Comment:
I suspect the type annotation should be `List[Link]` since an empty list is already falsey.

Reply:
Thanks you are right.